Source/Scripts/Sprocs/user_usp_UPDATE_Users.sql

```diff
@@ -0,0 +1,33 @@
+CREATE PROCEDURE [user].[usp_UPDATE_User_ByID]
+	@UserID int,
+	@Username varchar(50),
+	@Balance int
+AS
+BEGIN
+	BEGIN TRY
+		BEGIN TRANSACTION
+		UPDATE Users
+		SET Username = @Username, Balance = @Balance
+		WHERE UserID = @UserID
+
+		COMMIT TRANSACTION
+	END TRY
+	BEGIN CATCH
+		IF XACT_STATE() <> 0
+			BEGIN 
+				ROLLBACK TRANSACTION
+				PRINT('An error has occurred. The update transaction has been rolled back.')
+			END;
+		THROW
+	END CATCH
+
+	SELECT
```

Comment:
Remove this select, this sproc shouldn't return anything.

You should return the user as they exist in the Db now, but the Db does not programmatically generate any values that you yourself don't already have in code.